packages/airnode-utilities/test/setup/init/set-define-property.ts

```diff
@@ -0,0 +1,18 @@
+// Sometimes Jest can have issues when trying to spy on an internal module. Import
+// and use this function as a workaround at the top of your test.
+//
+// Credit: https://github.com/facebook/jest/issues/6914#issuecomment-654710111
```

Comment:
I actually wonder if we still need this file 🤔

Reply:
I'll try removing it...

Reply:
It appears that it is not necessary in this package.

Reply:
or any package except airnode-node :muscle: